h2o-k8s/src/main/java/water/k8s/KubernetesEmbeddedConfigProvider.java

```diff
@@ -1,20 +1,20 @@
 package water.k8s;
 
-import fi.iki.elonen.NanoHTTPD;
 import water.H2O;
 import water.init.AbstractEmbeddedH2OConfig;
 import water.init.EmbeddedConfigProvider;
+import water.init.NetworkInit;
```

Comment:
Unused?

Reply:
There was one more. Removed in https://github.com/h2oai/h2o-3/pull/5134/commits/11706dd558cec20d3df29ef3140c9172f0625a98 .